package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@risingstack/react-easy-state",
-  "version": "6.1.3",
+  "version": "0.0.0-development",
```

Comment:
Why did you rewrite this to a new version? We had a vote about this on Slack and we decided to continue versioning from the current version (6.1.3). This version is already published to npm with an alpha tag.

Reply:
This is a semantic-release configuration requirement. From now on sematic-release will be responsible for the versioning, and this version field will be constant.